lib/workers/pr/body/updates-table.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { BranchConfig, BranchUpgradeConfig } from '../../types';
+import { getPrUpdatesTable } from './updates-table';
+
+describe('workers/pr/body/updates-table', () => {
+  it('checks results for getPrUpdatesTable', () => {
+    const configObj: BranchConfig = {
+      branchName: 'some-branch',
+      upgrades: [],
+      prBodyColumns: ['Package', 'Type', 'Update', 'Change', 'Pending'],
+    };
+    const upgrade0: BranchUpgradeConfig = { branchName: 'some-branch' };
+    upgrade0.prBodyDefinitions = {
+      Package: '{{{depNameLinked}}}',
+      Type: '{{{depType}}}',
+      Update: '{{{updateType}}}',
+      'Current value': '{{{currentValue}}}',
+      'New value': '{{{newValue}}}',
+      Change: 'All locks refreshed',
+      Pending: '{{{displayPending}}}',
+      References: '{{{references}}}',
+      'Package file': '{{{packageFile}}}',
+    };
+    upgrade0.updateType = 'lockFileMaintenance';
+    const upgrade1: BranchUpgradeConfig = { branchName: 'some-branch' };
+    upgrade1.prBodyDefinitions = {
+      Package: '{{{depNameLinked}}}',
+      Type: '{{{depType}}}',
+      Update: '{{{updateType}}}',
+      'Current value': '{{{currentValue}}}',
+      'New value': '{{{newValue}}}',
+      Change:
+        "[{{#if displayFrom}}`{{{displayFrom}}}` -> {{else}}{{#if currentValue}}`{{{currentValue}}}` -> {{/if}}{{/if}}{{#if displayTo}}`{{{displayTo}}}`{{else}}`{{{newValue}}}`{{/if}}]({{#if depName}}https://renovatebot.com/diffs/npm/{{replace '/' '%2f' depName}}/{{{currentVersion}}}/{{{newVersion}}}{{/if}})",
+      Pending: '{{{displayPending}}}',
+      References: '{{{references}}}',
+      'Package file': '{{{packageFile}}}',
+    };
```

Comment:
```suggestion
    const upgrade1: BranchUpgradeConfig = {
        branchName: 'some-branch',
        prBodyDefinitions = {
        Package: '{{{depNameLinked}}}',
        Type: '{{{depType}}}',
        Update: '{{{updateType}}}',
        'Current value': '{{{currentValue}}}',
        'New value': '{{{newValue}}}',
        Change:
          "[{{#if displayFrom}}`{{{displayFrom}}}` -> {{else}}{{#if currentValue}}`{{{currentValue}}}` -> {{/if}}{{/if}}{{#if displayTo}}`{{{displayTo}}}`{{else}}`{{{newValue}}}`{{/if}}]({{#if depName}}https://renovatebot.com/diffs/npm/{{replace '/' '%2f' depName}}/{{{currentVersion}}}/{{{newVersion}}}{{/if}})",
        Pending: '{{{displayPending}}}',
        References: '{{{references}}}',
        'Package file': '{{{packageFile}}}',
      }
    };
```

please simplify object declarations

Reply:
done

Reply:
https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting 🤗

Reply:
i know of such option, i just dont have permission to do it here :/